### PR TITLE
Reformat sbt files

### PR DESCRIPTION
**Pull request by ArturGajowy**
## Overview
Title says it all :)

### JIRA issue:
n/a

### Checklist
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### CODE REVIEW

File: project/Detector.scala [hunk 1]
```diff
@@ -95,51 +98,50 @@ object Detector {
...
 
       val likeSet = Set(DefaultRedhatVariants: _*) + i
       LinuxRelease(i, version, likeSet)
     }
   }
 
-  private def normalizeOsReleaseValue(value: String): String = { // Remove any quotes from the string.
+  private def normalizeOsReleaseValue(value: String): String = // Remove any quotes from the string.
     value.trim.replace("\"", "")
-  }
```
Comment (KentShikama) [inline project/Detector.scala]: Removed braces

File: project/CompilerSettings.scala [hunk 2]
```diff
@@ -26,10 +26,8 @@ object CompilerSettings {
       "-feature",
       "-language:_",
       "-unchecked",
-      /*
-       * With > 16: [error] invalid setting for -Ybackend-parallelism must be between 1 and 16
-       * https://github.com/scala/scala/blob/v2.12.6/src/compiler/scala/tools/nsc/settings/ScalaSettings.scala#L240
-       */
+      //With > 16: [error] invalid setting for -Ybackend-parallelism must be between 1 and 16
+      //https://github.com/scala/scala/blob/v2.12.6/src/compiler/scala/tools/nsc/settings/ScalaSettings.scala#L240
```
Comment (KentShikama) [inline project/CompilerSettings.scala]: Block comment changed to line comment

File: project/BNFC.scala [hunk 3]
```diff
@@ -81,23 +87,28 @@ object BNFC {
...
+        cleanDocs      := cleanDir(bnfcDocDir.value),
+        generateDocs   := bnfcFiles(bnfcGrammarDir.value).foreach { (f: File) =>
+          bnfcGenerateLaTeX(f, bnfcDocDir.value)
         }
-      },
-      cleanDocs      := cleanDir(bnfcDocDir.value),
-      generateDocs   := bnfcFiles(bnfcGrammarDir.value).foreach { (f: File) =>
-        bnfcGenerateLaTeX(f, bnfcDocDir.value)
-      }))
+        // format: on
```
Comment (KentShikama) [inline project/BNFC.scala]: Format off/on comments added

File: project/GrpcMonixGenerator.scala [hunk 4]
```diff
@@ -73,16 +80,20 @@ class GrpcMonixGenerator(override val params: GeneratorParams)
...
+      )
+      .print(service.methods) {
+        case (printer, method) =>
+          printer.add(s".addMethod(${method.descriptorName})")
       }
       .add(".build()")
       .outdent
 
-  private[this] def serviceMethodSignature(method: MethodDescriptor) = {
+  private[this] def serviceMethodSignature(method: MethodDescriptor) =
```
Comment (KentShikama) [inline project/GrpcMonixGenerator.scala]: Braces removed

File: project/Rholang.scala [hunk 5]
```diff
@@ -3,37 +3,44 @@ import Keys._
...
+      resources: File
+  ): Seq[File] = {
     import scala.sys.process._
 
-    val command = 
-      s"java -jar ${jar.getAbsolutePath()} ${rhoSourceDir.getAbsolutePath()} ${srcManaged.getAbsolutePath()} ${resources.getAbsolutePath()}"
-      
+    val command =
+      s"java -jar ${jar.getAbsolutePath()} ${rhoSourceDir.getAbsolutePath()} ${srcManaged
+        .getAbsolutePath()} ${resources.getAbsolutePath()}"
```
Comment (KentShikama) [inline project/Rholang.scala]: I didn't know this was safe.

Comment (ArturGajowy) [reply project/Rholang.scala]: Neither did I :)

File: project/Detector.scala (hunk 1 above)
Comment (ArturGajowy) [reply project/Detector.scala]: Those were automatic

File: project/CompilerSettings.scala (hunk 2 above)
Comment (ArturGajowy) [reply project/CompilerSettings.scala]: This I changed myself, the formatted version was ugly otherwise.

File: project/BNFC.scala (hunk 3 above)
Comment (ArturGajowy) [reply project/BNFC.scala]: Added those to retain the align. Opened an issue in scalafmt for that.

File: project/GrpcMonixGenerator.scala (hunk 4 above)
Comment (ArturGajowy) [reply project/GrpcMonixGenerator.scala]: Automatic as well :)

**Comment by ArturGajowy** [comment]
Thx for the very vigilant review @KentShikama! :)

**Pull request merged**